pulsar/managers/util/cli/__init__.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, code_dir='lib'):
         """
         """
         def __load(module_path, d):
-            module_pattern = join(join(getcwd(), code_dir, *module_path.split('.')), '*.py')
+            module_pattern = join(join(dirname(__file__), module_path.split('.')[-1]), '*.py')
```

Comment:
I think the implementation of the pulsar variant and the Galaxy variant have diverged.
I think you could copy https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/jobs/runners/util/cli/__init__.py over and that should fix it too ? In either case we should unify these two before we diverge further.

Might even be worth to delete the Galaxy variant and only keep the pulsar code.

Reply:
related https://github.com/galaxyproject/galaxy/pull/10189